src/projects/project-activity.html

```diff
@@ -61,30 +57,17 @@
         padding-left: 0.3em;
         vertical-align: bottom;
       }
-      .info{
-        margin-top: 22px;
-      }
-      .container.false {
-        height: 1.3em;
-        overflow: hidden;
-        text-align: center;
-      }
-      .container.true {
-        height: auto;
-      }
-      .hovering.false {
-        text-align: center;
-      }
-      .collapsible-content.false {
-        visibility: hidden;
-      }
       .readme {
         padding: 0.75em;
       }
       .repeat {
         display: inline-block;
         padding: 6px;
       }
+      .container {
+        text-align: center;
+        margin-top: 1em;
```

Comment:
Can you also add a 1 em margin to the left and right? To ensure that the contributors never get too close to the edge.